Source/XUnitExtensions/Lit/ExitCommand.cs

```diff
@@ -14,7 +14,7 @@ public class ExitCommand : ILitCommand {
 
   public (int, string, string) Execute(ITestOutputHelper? outputHelper, TextReader? inputReader, TextWriter? outputWriter, TextWriter? errorWriter) {
     var (exitCode, output, error) = operand.Execute(outputHelper, inputReader, outputWriter, errorWriter);
-    return (exitCode != expectedExitCode ? 1 : 0, output, error);
+    return (exitCode != expectedExitCode ? 1 : 0, output, error + $"\nMoreover the expected exit code was {expectedExitCode} but got {exitCode}");
```

Comment:
This is manually tested and was useful to understand why my original tests with %exits-with 1 was failing without indication about what was wrong.